src/Observable.ts

```diff
@@ -17,7 +17,7 @@ export class Observable<T> extends BaseObservable<T> {
         }
 
         if (values[Symbol.iterator]) {
-            return new Observable<T>(({next, error}) => {
+            return new Observable<T>(({next, error, complete}) => {
```

Comment:
używaj proszę observer.next() zamiast destrukturyzowania obiektu observera, bo co prawda nasza implementacja ma zafiksowany kontekst, to jeżeli coś będzie używać `this` w tym next, to nie zadziała, bo wywołujesz funkcję bez kontekstu. #justJavaScriptThings